apollo-client/src/main/java/com/ctrip/framework/apollo/monitor/internal/model/CounterModel.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2022 Apollo Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+package com.ctrip.framework.apollo.monitor.internal.model;
+
+import com.ctrip.framework.apollo.monitor.internal.util.MeterType;
+
+/**
+ * @author Rawven
+ */
+public class CounterModel extends SampleModel {
```

Comment:
Can we use api com.dianping.cat.Cat#logMetricForCount(java.lang.String) and add it to tracer?

i.e the data will go through

```mermaid
graph
Tracer --> |logXxx|MessageProducer-OurCompositeProducer
MessageProducer-OurCompositeProducer --> |logXxx|ClientMonitorMessageProducer
MessageProducer-OurCompositeProducer --> |logXxx|CatMessageProducer
CatMessageProducer --> |invoke|CatApi
ClientMonitorMessageProducer --> |ApolloClientMetricEvent|List-of-MetricsCollector
*

Reply:
completed